crates/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -409,7 +409,9 @@ impl PathEnvironment<Original> {
         temp_dir: impl AsRef<Path>,
     ) -> Result<Self, EnvironmentError2> {
         let env_dir = dot_flox_path.as_ref().join(ENVIRONMENT_DIR_NAME);
+        tracing::debug!(?env_dir, "attempting to open environment directory");
         if !env_dir.exists() {
+            eprintln!("dir doesn't exist");
             Err(EnvironmentError2::EnvNotFound)?;
```

Comment:
suggestion: use logger

`tracing` is a broader concept than `log`
our logging to stderr is only handling tracing events that are [explicitly tagged as "log"](https://github.com/flox/flox/blob/3a21bf387792282bda2c2b1bfcd8cc0c463561a5/crates/flox/src/utils/logger.rs#L86-L88)

```suggestion
        log::debug!("attempting to open environment directory {env_dir:?}");
        if !env_dir.exists() {
            log::debug!("dir doesn't exist");
```

Reply:
I think our logging system is pretty strange, but that's a conversation for another day. I think all logging should be based on tracing and user-facing output should just be handled with `println!`.